docs/strict-typescript-api.md

```diff
@@ -0,0 +1,197 @@
+---
+id: strict-typescript-api
+title: Strict TypeScript API (opt in)
+---
+
+:::tip New in 0.80!
+
+The **Strict TypeScript API** is a new, opt-in, mode for using React Native with TypeScript, providing stronger type guarantees aligned with our direction for a stable React Native API. See the **announcement post (incoming)**.
+
+:::
+
+The Strict TypeScript API is a preview of our future, stable JavaScript API for React Native.
+
+Specifically, this is a new set of TypeScript types for the `react-native` npm package, available from 0.80 onwards. These enable stronger and more futureproof type accuracy, and will be a one-time breaking change.
```

Comment:
IMHO this is not entirely true.

What if we remove a lean core a package next month? That will also be a breaking change.

I think this should be reworded so that it's not a "one-time" breaking change but more on the sense of "fewer" breaking changes

Reply:
Ah yeah, I'm referring specifically to the initial swap-over being a breaking change for existing apps. This is constant.

If I move this around in the sentence it should work!